e2e/support/helpers/e2e-native-editor-helpers.ts

```diff
@@ -0,0 +1,20 @@
+export function nativeEditor() {
```

Comment:
This because we're using the same component there now too. I think it makes sense to not have the text stick to the border.

But it does go against my goal of not having any UX changes. This not caught by an Loki tests, so I think it's fine to leave it in.